control/control.SLES.xml

```diff
@@ -115,6 +115,9 @@ textdomain="control"
                 <packages>sblim-sfcb</packages>
             </service>
         </services_proposal>
+
+        <!-- FATE: #319716: Automatic Update of YaST at beginning of the installation -->
+        <!-- <self_update_url>http://update.opensuse.org/update.dud</self_update_url> -->
```

Comment:
Yes, because we don't have a definitive URL yet and I prefer a comment instead using a fake URL as the PBI suggested.